<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PendingOperationsDialog</class>
 <widget class="QDialog" name="PendingOperationsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Pending File Operations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QTreeWidget" name="opTreeWidget">
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Completed</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Operation</string>
         </property>
        </column>
        <item>
         <property name="text">
          <string>test</string>
         </property>
         <property name="text">
          <string>34</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>test</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="cancelBtn">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="cancelAllBtn">
        <property name="text">
         <string>Cancel All</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PendingOperationsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PendingOperationsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
